### DISCUSSION TITLE
New Meeting Does Not Display in TSML UI (v.3.14.2.)

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#794] [Q&A] [answered]
I hate to duplicate issues since this is related to #779 and #773, but in this instance, it's not just happening as the as the Admin, I also don't see changes on TSML UI on iOS Safari or other Windows-based PCs using Firefox. 

This is with TSML v.3.14.2.

I added a new meeting directly in WordPress. I was currently using the Legacy UI. I checked it and it is there. 
Then I switched to TSML UI and the meeting does not show up. It's there in the backend, but not the front. 

I then went to my staging site and made sure TSML UI was the default. I added the same meeting in the backend and got this error:

Warning: array_merge(): Expected parameter 2 to be an array, bool given in /public_html/wp-content/plugins/12-step-meeting-list/includes/functions.php on line 967

Warning: Invalid argument supplied for foreach() in/public_html/wp-content/plugins/12-step-meeting-list/includes/functions.php on line 990

This may or may not be related. I tried to enter again and same result. So I trashed the meeting. Purged cache just in case. Switched to Legacy UI. Successfully added meeting. Saw it was there and displayed fine in Legacy UI. Switched to TMSL UI and it's not showing. I checked on my phone (safari) and checked on laptop -- not there for either.

**Comment by 97eighty** [comment, marked as answer]
@Grateful-Viv Which versions of PHP are you using on your Production and Staging sites?

**Reply by Grateful-Viv** [reply]
Both are using 7.4.29

And an hour later I checked and the meeting is being displayed now.

**Reply by joshreisner** [reply]
btw @brianw-area24 's changes are not in the plugin yet -- they are in `develop` but not `main` -- should be deployed in `3.14.3`

**Reply by 97eighty** [reply]
@joshreisner Thanks. @Grateful-Viv Let's wait until 3.14.3 is released to verify.

**Reply by Grateful-Viv** [reply]
Absolutely, no worries. And thank you.

**Reply by joshreisner** [reply]
just want to circle back and see if this problem was solved @Grateful-Viv ?

**Reply by Grateful-Viv** [reply]
Yes, @joshreisner all seems great now with respect to this issue. Thanks for circling back.
